libkbfs/data_types.go

```diff
@@ -941,3 +941,8 @@ func SessionInfoFromProtocol(session keybase1.Session) (SessionInfo, error) {
 		VerifyingKey:   verifyingKey,
 	}, nil
 }
+
+// NodeMetadata has metadata about a node needed for higher level operations.
+type NodeMetadata struct {
+	LastWriter keybase1.UID
```

Comment:
Didn't we talk about naming this `UnverifiedLastWriter`, to clarify that a modified client could fake this info?

Although one caveat to that, if the file is actually readable, then the block server would have enforced that the writer was really the one who wrote the block in the first place.